source/fundamentals/connection.txt

```diff
@@ -69,11 +69,20 @@ of the ``replicaSet`` parameter in the connection string.
 
    mongodb://host1:27017,host2:27017,host3:27017/?replicaSet=myRs
 
+
 .. note::
 
    Although it is possible to connect to a replica set deployment by
    providing only a single host, you should provide the driver with the
-   full list to ensure that it is able to connect even if one host fails.
+   full list to ensure that it is able to connect in the event the specified host is unavailable.
```

Comment:
**suggestion:** Tighten the language somewhat using second person and imperative mood.
```suggestion
   You only need to provide a single host to connect to a deploy replica set deployment. 
   To ensure connectivity when the specified host is unavailable, provide the driver with
   the full list of hosts
```